lib/archethic/oracle_chain/scheduler.ex

```diff
@@ -64,8 +84,10 @@ defmodule Archethic.OracleChain.Scheduler do
         index = chain_size(summary_date)
         Logger.info("Oracle Scheduler: Scheduled during init - (index: #{index})")
 
-        {:ok, :idle,
+        {:idle,
          %{
+           # server data must have old key
+           args: nil,
```

Comment:
what is this variable ?

Reply:
With introduction, of PubSub.register_for_node_up, process_data modification were required.
 If  `archethic_up` is `nil` we pass the args of `init(args)` into the process_data of FSM as `%{args: args}`,
Where it waits for `:node_up` as soon as message receives, start scheduler by reading data which has key :args.

Reply:
You can set the interval in the state even without archethic_up event. This should be common.